src/middlewared/middlewared/plugins/service_/services/iscsitarget.py

```diff
@@ -23,3 +23,11 @@ async def before_stop(self):
         if osc.IS_FREEBSD:
             with contextlib.suppress(IndexError):
                 sysctl.filter("kern.cam.ctl.ha_peer")[0].value = ""
```

Comment:
isnt this broken in sysctl module and should use subprocess?

Reply:
Why? I don't remember this

Reply:
I think it should be, we changed that usage in ctld.py, but interestingly I think stopping iscsi service works, confirming.

Reply:
It doesn't, that sysctl I think is set/exists on HA systems so that raised an index error and I thought it worked. I'll correct this usage